src/components/IllustrationItem/IlloImage.jsx

```diff
@@ -4,20 +4,27 @@ import { colors, flex } from '../../theme';
 import styles from './styles.module.css';
 import IlloHover from './IlloHover';
 import useIllustrationFallback from '../../hooks/useIllustrationFallback';
+import atoms from '../../atoms';
 
 export default function IlloImage({
   id,
   illoName,
   onClick,
   hoverState,
   setHoverState,
-  isArchive,
 }) {
-  const imgContainer = cls(styles.imgContainer, flex.flex, flex.flexCenter, {
-    [colors.bgSlate700]: !isArchive,
-  });
+  const imgContainer = cls(
+    styles.imgContainer,
+    flex.flex,
+    flex.flexCenter,
+    colors.bgSlate700,
+  );
+
+  const { lastGeneratedDate } = atoms.useRecoilValue(
```

Comment:
If were only using the details for the preview, it should be called from the useIllustrationFallback hook.